financial-policy.md

```diff
@@ -0,0 +1,69 @@
+
+Any attempts to dispute a valid membership charge without a cancellation request will be treated as a cancellation request. The member will be prevented from obtaining a new membership unless the Treasurer approves.
+
+### Monetary Donations:
+From time to time, we will receive monetary donations from a member or third-party charity (Amazon Smile). All donations will be deposited into our general fund for space use unless otherwise directed by the donor. In this case, we will attempt to use the funds as directed and as circumstances permit.
+
+### Grants:
+From time to time, we will receive grants from organizations. All grants will be spent per the grant documentation. Should no specific use be called out, funds will be deposited into our general fund for space use.
+
+## How we Spend our Money
+
+### Board Approved Funds (AKA: Project Funds):
+#### Approval:
+Any member in good standing can submit a project request to the Board. Should these requests require funds from the general balance, the Treasurer will confirm if the funds are available, and the Board will vote to approve the project and requested funds.
+
+Once approved, an official Project Fund will be created, and the member will be considered the Project Owner. Project Ownership can be transferred between members in good standing as long as both are aware and have confirmed with the Treasurer.
+
+#### Updates:
+The Treasurer will provide a monthly summary of all active Project Funds and their remaining balances. The Treasurer will also check in with the Project Owner every ~3 months to ensure the project is still active and funds are still required. Check-in can be skipped at the Treasurer's discretion.
+
+#### Project Closure:
+A project will be considered closed when all funds have been dispersed or when the Project Owner has informed the Treasurer that the funds are no longer required. The Treasurer will report any Project closures to the Board monthly and any funds returning to the general balance.
+
+#### Abandoned Project:
+A project will be considered abandoned when no funds have been dispersed after 90 days and no update has been provided to the Treasurer. The Treasurer will report any Abandoned Projects to the Board monthly and any funds returning to the general balance.
+
+### Recurring Operating Funds (AKA: Manager Budgets):
+#### Approval:
+When requested by an acting Manager, the Board can approve a recurring operating fund that a Manager can use at their discretion for their area of business. The Treasurer will confirm if funds are available in the general balance.
+
+#### Updates:
+The Treasurer will provide a monthly summary of all active Manager Budgets and their spending funds. The Treasurer will also check in with the acting Manager every ~ six months to ensure the funding is appropriate and will help guide a request to increase or decrease funds. Check-in can be skipped at the Treasurer's discretion.
+
+### Vending Machine Pool:
```

Comment:
Do we need to specify the specific pool here or just that we can make and operate pools? I just don't want you to have to update a whole financial policy when one new pool gets approved.